test/server/dom-shims.js

```diff
@@ -41,3 +42,16 @@ describe(`customElement with shadowDom`, function () {
     expect(elem.el.innerHTML).to.equal(`<style>color: blue;</style><div class="foo"><p>Hello</p></div>`);
   });
 });
+
+describe(`appending customElements to DOM`, function () {
+  it(`calls connectedCallback on customElements when using appendChild`, async function () {
+    const elem = document.createElement(`my-app`);
+    const connectedCallbackStub = sinon.stub(elem, `connectedCallback`);
+
+    document.createElement(`div`).appendChild(elem);
```

Comment:
it's weird making a `div` for this, but it does correctly test the behavior.

I would have preferred to use `document.appendChild` or `document.body.appendChild`, but to do that correctly I will need to submit a PR to `html-element` to make the shim more correct